src/bsoncxx/array/view.cpp

```diff
@@ -59,7 +59,7 @@ view::const_iterator& view::const_iterator::operator++() {
     bson_iter_next(&i);
 
     if (!bson_iter_next(&i)) {
-        _element = element{nullptr, 0, 0};
+        _element = element{};
```

Comment:
Just an obvious cleanup.

Reply:
Maybe it is even possible to write element = {}; ?

Reply:
Maybe. I'm not too inclined to change it though.